retry/retry.go

```diff
@@ -60,7 +60,7 @@ func (r Retry) Do(ctx context.Context, f func(context.Context) error, desc strin
 	if err == nil || r.maxRetry == 0 {
 		return err
 	} else if errors.As(err, &nonRetryableError) {
-		r.logger.Debug("received non retryable error")
+		r.logger.Debug("received non-retryable error", "method", desc)
```

Comment:
Should other instances of `description` be changed to `method` for consistency?

Reply:
and should these be included in all logs in this function?

Reply:
Oh, I didn't notice that we had logged description later.  I didn't think `description` makes sense without the context of this method (as in, if you see this log and don't know about how we do retries, you don't know what this is a description of).

But `method` isn't really all that better when I think about it and this really should have been a separate PR anyway, so I'll change it to `description` and that can be some future evaluation.